packages/attestation-service/src/sms/nexmo.ts

```diff
@@ -156,6 +159,11 @@ export class NexmoSmsProvider extends SmsProvider {
     if (matchingNumber !== undefined) {
       return matchingNumber.phoneNumber
     }
-    return this.nexmoNumbers[0].phoneNumber
```

Comment:
```
const matchingNumber = this.nexmoNumbers.find((number) => number.code === countryCode)
```

^ I'd think you also need to ensure for the US case, this would this resolve to the TFN and not another US number

```
if (countryCode == "US") {
 matchingNumber = this.nexmoNumbers.find((number) => number.code === countryCode && number.type == this.tollFreeType)
}
```

Reply:
Good catch! Thanks